src/events/MessageCreateEvent.ts

```diff
@@ -2,19 +2,25 @@ import { createEmbed } from "../utils/functions/createEmbed";
 import { BaseEvent } from "../structures/BaseEvent";
 import { Event } from "../utils/decorators/Event";
 import i18n from "../config";
-import { Message, User } from "discord.js";
+import { ChannelType, Message, User } from "discord.js";
 
 @Event<typeof MessageCreateEvent>("messageCreate")
 export class MessageCreateEvent extends BaseEvent {
     public execute(message: Message): Message | undefined {
         this.client.debugLog.logData("info", "MESSAGE_CREATE", [
             ["ID", message.id],
             ["Guild", message.guild ? `${message.guild.name}(${message.guild.id})` : "DM"],
-            ["Channel", message.channel.type === "DM" ? "DM" : `${message.channel.name}(${message.channel.id})`],
+            [
+                "Channel",
+                message.channel.type === ChannelType.DM
+                    ? ChannelType.DM.toString()
+                    : `${message.channel.name}(${message.channel.id})`
+            ],
```

Comment:
You don't need to change the `DM` string with enum value on the ternary true statement